src/general/MainMenu.cs

```diff
@@ -277,6 +277,14 @@ public override void _Notification(int notification)
             GD.Print("Main window close signal detected");
             Invoke.Instance.Queue(QuitPressed);
         }
+        else if (notification == NotificationTranslationChanged)
+        {
+            if (SteamHandler.Instance.IsLoaded)
+            {
+                storeLoggedInDisplay.Text = TranslationServer.Translate("STORE_LOGGED_IN_AS")
+                    .FormatSafe(SteamHandler.Instance.DisplayName);
```

Comment:
This is the same as in line `595`, might be best to put this in its own method (since calling the whole `UpdateStoreVersionStatus` is too overkill).

Reply:
I moved it to a method called UpdateSteamLoginText and updated the two methods to call it